apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js

```diff
@@ -268,6 +265,28 @@ describe('The JSDebugger component', () => {
   });
 });
 
+function moveMouse(xTarget, yTarget) {
+  const event = document.createEvent('MouseEvents');
+  event.initMouseEvent(
```

Comment:
Nit: [`initMouseEvent` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/initMouseEvent).

Reply:
Leaving this as-is for two reasons:
1. new MouseEvent isn't supported in phantom js yet
2. this test (prior to the Enzyme 3 update) is broken in ChromeHeadless (and therefore likely also broken in Chrome) due to the way the events in other places are set up.
Due to #2, this only works in phantom js. I'll create a future item to make this cross-browser compatible. Not doing this work as part of this change.